marlowe-dashboard-client/webpack.config.js

```diff
@@ -18,19 +18,25 @@ module.exports = {
         https: true,
         stats: "errors-warnings",
         proxy: {
-            "/api": {
+            "/pab/api": {
```

Comment:
@shlevy we need to modify the Traefik configuration accordingly, not sure where is that located.

Reply:
@hrajchert In the current state everything is served from the PAB, so traefik just sends everything there. We'll only have to worry about routes once we have the marlowe-run-server in the mix.

Reply:
Ah I misread the PR, it looks like that's what's happening here.

Will it make sense to deploy this just using all of the mock stuff from the PAB, or is it only something we can deploy with a proper WBE, node, etc.?

Reply:
Yes, the idea of this PR is to bring marlowe-run-server into the mix. I'm not too worried about the deployment as there are other things that need to happen for this to properly work, but yes, we are going to need the WBE and node in the deployment relatively soon.